appassembler-maven-plugin/src/main/java/org/codehaus/mojo/appassembler/daemon/yajsw/YetAnotherJavaServiceWrapperDaemonGenerator.java

```diff
@@ -0,0 +1,317 @@
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * @author Sandra Parsick
+ * @version $Id$
+ * @plexus.component role-hint="yajsw"
+ */
+public class YetAnotherJavaServiceWrapperDaemonGenerator
+    extends AbstractLogEnabled
+    implements DaemonGenerator
+{
+
+    private static List<String> CORE_LIBS = new ArrayList<String>()
+    {
+        {
+            add("commons/commons-cli-1.4.jar");
+            add("commons/commons-collections-3.2.2.jar");
+            add("commons/commons-configuration2-2.3.jar");
+            add("commons/commons-io-2.6.jar");
+            add("commons/commons-lang3-3.8.1.jar");
+            add("commons/commons-lang-2.6.jar");
+            add("commons/commons-logging-1.1.jar");
+            add("commons/commons-text-1.6.jar");
+            add("commons/commons-vfs2-2.2.jar");
+            add("jna/jna-5.3.1.jar");
+            add("jna/jna-platform-5.3.1.jar");
+            add("netty/netty-all-4.1.36.Final.jar");
+            add("yajsw/ahessian.jar");
+        }
+    };
+
+    private static List<String> EXTENDED_LIBS = new ArrayList<String>()
+    {
+        {
+            add("velocity/velocity-1.7.jar");
+            add("yajsw/hessian4.jar");
+            add("groovy/groovy-2.5.7.jar");
+            add("groovy/groovy-patch.jar");
+        }
+    };
+
+    private static List<String> BAT_SCRIPTS = new ArrayList<String>()
+    {
+        {
+              add("installService.bat");
+              add("runConsole.bat");
+              add("startService.bat");
+              add("stopService.bat");
+              add("uninstallService.bat");
+              add("setenv.bat");
+              add("wrapper.bat");
+        }
+    };
+
+    private static List<String> SHELL_SCRIPTS = new ArrayList<String>()
+    {
+        {
+            add("installDaemon.sh");
+            add("runConsole.sh");
+            add("startDaemon.sh");
+            add("stopDaemon.sh");
+            add("uninstallDaemon.sh");
+            add("setenv.sh");
+            add("wrapper.sh");
+            add("installDaemonNoPriv.sh");
+            add("uninstallDaemonNoPriv.sh");
+            add("startDaemonNoPriv.sh");
+            add("stopDaemonNoPriv.sh");
+        }
+    };
+
+    private static List<String> GROOVY_SCRIPTS = new ArrayList<String>()
+    {
+        {
+            add("trayMessage.gv");
+        }
+    };
+
+    private static List<String> TEMPLATES = new ArrayList<String>()
+    {
+        {
+            add("daemon.vm");
+        }
+    };
+
+    @Override
+    public void generate( DaemonGenerationRequest request ) throws DaemonGeneratorException
+    {
+        Daemon daemon = request.getDaemon();
+
+        File outputDirectory = new File( request.getOutputDirectory(), daemon.getId() );
+
+        writeWrapperConfFile(outputDirectory, daemon, request);
+
+        writeScripts(outputDirectory);
+        writeLibraries(outputDirectory);
+
+
```

Comment:
Cosmetic: These empty lines could be removed.

Reply:
Thank you. Done